cmake/Dependencies.cmake

```diff
@@ -87,10 +87,10 @@ endif()
 # libjpeg-turbo
 ##################################################################
 if (BUILD_JPEG_TURBO)
-  find_package(JpegTurbo 1.5 REQUIRED)
-  include_directories(SYSTEM ${JpegTurbo_INCLUDE_DIR})
-  list(APPEND DALI_LIBS ${JpegTurbo_LIBRARY})
-  list(APPEND DALI_EXCLUDES libturbojpeg.a)
+  find_package(JPEG REQUIRED)
```

Comment:
Don't we need to enforce any version?

Reply:
How about using JPEG_TURBO_ROOT_DIR when user doesn't have JPEG installed, but have it compiled on the side?

Reply:
You can specify custom location like this:
cmake -DJPEG_INCLUDE_DIR=/home/janton/libjpeg-turbo-1.5.3/install/include/ -DJPEG_LIBRARY=/home/janton/libjpeg-turbo-1.5.3/install/lib/libjpeg.so ..

Reply:
Version enforced in last commit